app/models/embedded_ansible_worker/object_management.rb

```diff
@@ -53,4 +54,91 @@ def ensure_host(provider, connection)
       :variables => {'ansible_connection' => "local"}.to_yaml
     ).id
   end
+
+  CONSOLIDATED_PLUGIN_PLAYBOOKS_TEMPDIR = Pathname.new("/var/lib/awx_consolidated_source").freeze
```

Comment:
Can this live either down with the method that uses it or up at the top of the class?